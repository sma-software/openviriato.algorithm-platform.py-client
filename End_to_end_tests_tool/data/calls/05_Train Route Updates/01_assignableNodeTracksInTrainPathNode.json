{
    "restResource": "/nodes/{nodeId}/node-tracks",
    "httpMethod": "GET",
    "urlParameters": [ "=sepTrain:trainPathNodes[2].nodeId=" ],
    "queryParameters": [
        {
            "key": "assignableForTrainPathNodeId",
            "value": "=sepTrain:trainPathNodes[2].id=",
            "optional": false
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetAssignableNodeTracksForTrainPathNode(nodeID, trainPathNodeID, stopStatus)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Int64", "System.Nullable{SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.StopStatus}" ],
    "subcategory": "Get Assignable Node Tracks"
}
