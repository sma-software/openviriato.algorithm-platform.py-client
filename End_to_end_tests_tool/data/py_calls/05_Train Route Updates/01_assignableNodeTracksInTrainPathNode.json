{
  "PythonParameterMapping": {
    "node_id": "$.urlParameters.[0]",
    "train_path_node_id": "$.queryParameters.[0].value",
    "stop_status": "$.requestBody"
  }
}

