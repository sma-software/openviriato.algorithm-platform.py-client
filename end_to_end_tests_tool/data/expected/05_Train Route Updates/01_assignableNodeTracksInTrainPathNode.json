[
  {
    "id": 163,
    "code": "2",
    "debugString": "stationtrack:85AR_{StationTrack SID = 34140}"
  }
]